mne_bids_pipeline/_logging.py

```diff
@@ -80,8 +79,12 @@ def gen_log_kwargs(
 
     message = f' {message}'
 
-    script_path = pathlib.Path(os.environ['MNE_BIDS_STUDY_SCRIPT_PATH'])
-    step_name = f'{script_path.parent.name}/{script_path.stem}'
+    script_path = os.environ.get('MNE_BIDS_STUDY_SCRIPT_PATH')
```

Comment:
The env var gets set once we're running a processing script; it however run.py wants to log a message, this variable is not set, so we have to handle this